integration-tests/manager-integration-tests/manager-integration-tests-v1/README.md

```diff
@@ -0,0 +1,145 @@
+- ${manager.authentication.token} to use a token for communication with Manager
+- ${slack.channel.`<Channel name>`.name} to use actual channel name
+- ${slack.channel.`<Channel name>`.webhook.url} to use a WebHook URL for sending message to the specified channel
+- ${slack.token} to use a token for communication with Slack
+- ${topic.`<Topic name>`} to use a unique Kafka topic name
+- ${uuid.`<Uuid name>`} to use unique identifier. Useful to distinguish historical data produced by the same test for example.
+- ${webhook.site.token.`<Token name>`} to use a webhook.site token with specified alias
+
+## Test execution
+
+### Local testing
+
+If you deployed application locally (by [running manager and operator locally](../../../dev/README.md)) you can use [run-local-tests.sh](run-local-tests.sh) script to launch the tests with all local parameters set automatically.
+
+Script has 3 options:
+- `-t TAGS`  
+  Specify which tags to execute  
+  See also https://github.com/cucumber/cucumber-jvm/tree/main/junit-platform-engine#tags
+- `-p`  
+  Run tests in parallel
+- `-k`  
+  Keep created data, aka do not perform cleanup. Mostly used for local test run and debug.
+
+
+### Remote testing
+
+For remote testing you can execute tests using Maven command:
+
+```bash
+export OPENSHIFT_OFFLINE_TOKEN=<openshift-offline-token> # To obtain it, go to https://console.redhat.com/openshift/token
+mvn clean verify -Pcucumber -Devent-bridge.manager.url=<MANAGER_URL> -Dkeycloak.realm.url=<KEYCLOAK_URL> -Dbridge.client.id=<CLIENT_ID> -Dtest.credentials.file=localconfig.yaml
+# Example:
+# <CLIENT_ID>="cloud-services"
+# <KEYCLOAK_URL>="https://sso.redhat.com/auth/realms/redhat-external"
+```
+
+### Remote system access configuration
+
+To properly run the tests locally you need to specify environment variable to access webhook.site:
+- `WEBHOOK_SITE_TOKEN` - UUID to be used for tests interacting with webhook.site
+
+You can define this environment variable in localconfig.properties file or via the maven command, all environment variables are loaded before running the tests
+
+Check the [localconfig-example.properties](localconfig-example.properties) file for an example of how to use it.
+
+Also you need to create a localconfig.yaml file containing external system credentials and URLs for TNB framework. Check the [localconfig-example.yaml](localconfig-example.yaml) file for an example of how to use it.
+
+### Keycloak token authentication
+
+By Default test runs with those authentication parameters for Keycloak:
+
+```xml
+<bridge.client.id>kermit</bridge.client.id>
+<bridge.client.secret>N5TW1EfuIcQsplRsLXJ1aE3DZZMPN5ZH</bridge.client.secret>
+```
+
+You can update the parameters needs to configure in `integration-tests/manager-integration-tests/manager-integration-tests-v1/pom.xml` or via the maven command.
+
+### Performance testing
+
+TO-DO: this section is a copy from v1 until a real performance tests are created for v2.
```

Comment:
I think it's a typo :)

Reply:
Ah, of course, you are right :) Will fix.

Reply:
Fixed.